target/ppc/fpu_helper.c

```diff
@@ -3351,8 +3319,6 @@ void helper_xssubqp(CPUPPCState *env, uint32_t opcode,
     ppc_vsr_t t = *xt;
     float_status tstat;
 
-    helper_reset_fpstatus(env);
```

Comment:
There are more calls to `helper_reset_fpstatus` through `gen_reset_fpstatus`. We can remove them if the insns calls `gen_helper_float_check_status` (directly or with `gen_compute_fprf_float64`).